src/App.tsx

```diff
@@ -1,27 +1,53 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
 import { getNumbers } from './utils';
+import { Pagination } from './components/Pagination';
+
+
+let a = Array.from({length:5}, (_, index)=>index)
+console.log(a);
+
 
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 const items = getNumbers(1, 42).map(n => `Item ${n}`);
 
 export const App: React.FC = () => {
+  const [select, setSelect] = useState('5');
```

Comment:
The `select` state is initialized as a string. Ensure that it is always converted to a number when used in calculations, as seen in lines 23 and 24.